packages/main/src/components/AnalyticalTable/AnalyticalTable.stories.tsx

```diff
@@ -54,6 +61,22 @@ const columns = [
         </select>
       );
     }
+  },
+  {
+    id: 'actions',
+    Header: 'Actions',
+    accessor: '.',
+    width: 100,
+    canResize: false,
+    Cell: ({ cell, row }) => {
+      // console.log('This is your row data', row.original);
```

Comment:
Please uncomment or remove.

Reply:
this is intended to be there if someone reads the code :)